### PR TITLE
Feat/sidebar2

### PR DESCRIPTION
**Pull request by Minji1234**
I made rating system. When you click thumb emoticon it will store and render cool UI.

### CODE REVIEW

**Review by rachelg98 — CHANGES_REQUESTED** [review]
Currently it seems that the user can only rate a single answer to a question. Perhaps it would be nice if the user could rate answers individually  
Additionally, the answer ratings seem to be reset when the page is refreshed

**Comment by Minji1234** [comment]
I modified rating in AnswerView. If you click on rate up or rate down emoticon on the AnswerView and refresh the window, it will show your rating has been successfully reflected in the data.
**It shows your rated answer from the bottom position of the screen**

File: oga/frontend/src/containers/Profile/Profile.js [hunk 1]
```diff
@@ -278,7 +278,9 @@ const mapStateToProps = state => {
     return {
         userProfile: state.auth.profile,
         myQuestions: state.question.questions,
-        myAnswers: state.answer.answers
+        myAnswers: state.answer.answers,
+        counts_rating_up: state.rating.rating_up,
```
Comment (rachelg98) [inline oga/frontend/src/containers/Profile/Profile.js]: These two lines seem to generate an error in the Profile page.

**Review by rachelg98 — CHANGES_REQUESTED** [review]
Great job! :)
Perhaps it would be nice to display the number of likes and dislikes for each answer and have an Integer field for ratings instead of just a single boolean field. Otherwise an answer can only be rated a single time.

File: oga/frontend/src/containers/Profile/Profile.js (hunk 1 above)
Comment (rachelg98) [reply oga/frontend/src/containers/Profile/Profile.js]: Commented out these two lines for now

**Review by rachelg98 — APPROVED** [review]

**Pull request merged**